test/http_client_SUITE.erl

```diff
@@ -20,12 +20,19 @@
 -include_lib("common_test/include/ct.hrl").
 
 all() ->
+    [{group, fusco},
+     {group, gun}].
+
+groups() ->
+    [{fusco, [], tests()},
+     {gun, [], tests()}].
+
+tests() ->
     [get_test,
      no_pool_test,
      post_test,
      request_timeout_test,
-     pool_timeout_test
-    ].
+     pool_timeout_test].
```

Comment:
I don't think that change is necessary 🤷 
PS: also, I like being able to press `A` in vim and got to edit mode directly at the end of the line, in this case if I wanted to add a test I'd have to get to the end of the word with a movement relative to my current position... many more keystrokes 😛